docs/reference/commandline/service_update.md

```diff
@@ -154,6 +154,47 @@ $ docker service update --mount-rm /somewhere myservice
 myservice
 ```
 
+### Rolling back to the previous version of a service 
+
+Use the `--rollback` options to roll back to the previous version of the service.This will revert the service to the configuration that was in place before the most recent docker service update command.
```

Comment:
`--rollback options` should be `--rollback option`.

There is a missing space between `service.` and `This`.

Can you please wrap long lines of text? We don't seem to be particularly consistent about this, but I notice that mode lines in that file are wrapped at 80 characters.